readchar/readchar.py

```diff
@@ -21,7 +21,7 @@ def readkey(getchar_fn=None):
     if ord(c1) != 0x1b:
         return c1
     c2 = getchar()
-    if ord(c2) != 0x5b:
+    if ord(c2) not in (0x5b, 0x4f):
```

Comment:
Of course, we still can''t handle F 11, 12 because of the same concept as the ESC key problem... :(